sdk/core/core-rest-pipeline/src/retryStrategies/throttlingRetryStrategy.ts

```diff
@@ -28,28 +34,93 @@ function parseRetryAfterHeader(headerValue: string): number | undefined {
   }
 }
 
+/**
+ * Returns the number of milliseconds to wait based on "retry-after-ms" or "x-ms-retry-after-ms" header value.
+ * Returns undefined if there is no valid value.
+ * @internal
+ */
+function parseRetryAfterMSHeader(retryAfterMSHeaderValue: string | undefined): number | undefined {
+  const delayValueString = retryAfterMSHeaderValue;
+
+  if (delayValueString == null) {
```

Comment:
can a stricter equality check be used here? also is it checking for `null` when the type of `retryAfterMSHeaderValue` is `string | undefined`?

Reply:
Maybe `if(!delayValueString)` is enough?